db/schema.rb

```diff
@@ -454,6 +454,7 @@
     t.datetime "updated_at", null: false
     t.bigint "user_id"
     t.datetime "verified_at"
+    t.index ["user_id"], name: "index_email_authorizations_on_user_id"
```

Comment:
this looks like it's not supposed to sneak in :D

Reply:
oops. Probably after running migrations. What would be a good way to remote this? Do I just remove the line?

Reply:
yeah, just remove the line :)